test.js

```diff
@@ -3,20 +3,24 @@ import difflet from 'difflet';
 import m from './';
 
 const diff = difflet({indent: 2});
-
-test.cb(t => {
-	const expected = {
-		'elements/colors': {
-			style: true,
-			documentation: true
-		},
-		'elements/grid': {
-			style: true,
-			documentation: true
+const expected = {
+	elements: {
+		colors: {
+			path: 'src/elements/colors/',
```

Comment:
@lejoe is path redudant here given that `src` was already passed (as config) and `elements/colors/` is the "path" on the returned object?